src/igvfd/schemas/changelogs/ontology_term.md

```diff
@@ -1,5 +1,6 @@
 ## Changelog for *`ontology_term.json`*
 
+* Add calculated property `ontology`. (06/07/2023)
```

Comment:
add changelog to all the concrete classes, the actual ontologies

Reply:
Done